frontend/src/app/pages/dashboard/dashboard.component.html

```diff
@@ -0,0 +1,4 @@
+<h1 class="navbarTitle">{{title}}</h1>
```

Comment:
For the dashboard page, it would make sense to have 'Dashboard' as big text not in the navbar. See below example (their navbar is vertical but its the same idea).

![image](https://github.com/RyleyGG/apluschiever/assets/91753731/0bb18d43-3237-42df-9a12-ed45a7c3fa00)